docs/src/manual/tutorial.md

```diff
@@ -0,0 +1,773 @@
+    ─────────────────────────────────────────────────────────────────────
+    "AA"           -39           -15           978           970
+    "AS"           -43           -15           183           172
+    "B6"           -44           -14           335           310
+    "CO"           -55           -18           957           981
+    "DL"           -32           -17           701           730
+    "EV"           -40           -18           469           479
+    "F9"           -24           -15           277           275
+    "FL"           -30           -14           500           507
+    "MQ"           -38           -23           918           931
+    "OO"           -57           -33           380           360
+    "UA"           -47           -11           861           869
+    "US"           -42           -17           433           425
+    "WN"           -44           -10           499           548
+    "XE"           -70           -19           634           628
+    "YV"           -32           -11           72            54
+
+
+
+For each day of the year, count the total number of flights and sort in descending order:
+
+
+```julia
+Lazy.@as x flights begin
+    groupby(length, x, :DayofMonth)
+    sort(x, :length, rev = true)
+end
+```
+
+
+
+
+    Table with 31 rows, 2 columns:
+    DayofMonth  length
+    ──────────────────
+    28          7777
+    27          7717
+    21          7698
+    14          7694
+    7           7621
+    18          7613
+    6           7606
+    20          7599
+    11          7578
+    13          7546
+    10          7541
+    17          7537
+    ⋮
+    25          7406
+    16          7389
+    8           7366
+    12          7301
+    4           7297
+    19          7295
+    24          7234
+    5           7223
+    30          6728
+    29          6697
+    31          4339
+
+
+
+For each destination, count the total number of flights and the number of distinct planes that flew there
+
+
+```julia
+groupby(@NT(flight_count = length, plane_count = length∘union), flights, :Dest, select = :TailNum)
+```
+
+
+
+
+    Table with 116 rows, 3 columns:
+    Dest   flight_count  plane_count
+    ────────────────────────────────
+    "ABQ"  2812          716
+    "AEX"  724           215
+    "AGS"  1             1
+    "AMA"  1297          158
+    "ANC"  125           38
+    "ASE"  125           60
+    "ATL"  7886          983
+    "AUS"  5022          1015
+    "AVL"  350           142
+    "BFL"  504           70
+    "BHM"  2736          616
+    "BKG"  110           63
+    ⋮
+    "SJU"  391           115
+    "SLC"  2033          368
+    "SMF"  1014          184
+    "SNA"  1661          67
+    "STL"  2509          788
+    "TPA"  3085          697
+    "TUL"  2924          771
+    "TUS"  1565          226
+    "TYS"  1210          227
+    "VPS"  880           224
+    "XNA"  1172          177
+
+
+
+## Window functions
```

Comment:
I was surprised by this title... When someone says I think of something like "time window".
.